src/generator/index.tsx

```diff
@@ -127,7 +127,7 @@ export default async function renderMessages({ messages, channel, callbacks, ...
         {/* profiles */}
         <script
           dangerouslySetInnerHTML={{
-            __html: `window.$discordMessage={profiles:${await profiles}}`,
+            __html: `window.$discordMessage={profiles:${JSON.stringify(await profiles)}}`,
```

Comment:
Can also just not include the script if `ssr` is enabled? That way, there's no less bloat in the HTML. I think this script is needed just for client-side hydration.

```tsx
{!options.ssr && <script dangerouslySetInnerHTML={{...}}></script>}
```